src/lib.rs

```diff
@@ -199,6 +199,9 @@
 
 #![allow(dead_code)]
 
+#[macro_use]
```

Comment:
Nit: Since rkv is now on Rust 2018, macros can be import by `use crate::macro_foo`.

Reply:
Nice catch! Fixed in 9e147978d5752663c402c49c00abd0f86ce0887a.